packages/vuepress-theme-odyssey/global-components/Anatomy.vue

```diff
@@ -1,6 +1,14 @@
 <template>
-  <figure v-if="img" class="docskit-figure-anatomy">
-    <img :src="img" :alt="alt" />
+  <figure v-if="img" 
+      :class="{
+      'docskit-figure-anatomy': true,
+      [`is-figure-anatomy-filled`]: fill
+    }"
+  >
+    <div
+      v-html="require(`!html-loader!../../docs/.vuepress/public${img}`)"
```

Comment:
Suggest separating `${img}` and `public` with a slash and update `img` tags accordingly, as it'll be much easier to read.